src/Evolution/DiscontinuousGalerkin/Limiters/Minmod.cpp

```diff
@@ -88,15 +99,25 @@ bool minmod_limited_slopes(
     const auto& volume_and_slice_indices_d =
         gsl::at(buffer->volume_and_slice_indices, d);
 
+    // Compute slope of the linearized U from finite difference across first and
+    // last grid points. When using Gauss points, we also need to compute the
+    // the distance separating these grid points at it will be less than 2.0.
```

Comment:
`at it will` -> `as it will`

Reply:
👍 Made this fix and did a slight rephrasing of the first sentence as well